src/KeyworkerManagement/App.js

```diff
@@ -42,6 +42,14 @@ class App extends React.Component {
       return config;
     }, (error) => Promise.reject(error));
 
+    axios.interceptors.response.use((config) => {
+      if (config.status === 205) {
+        alert("There is a newer version of this website available, click ok to ensure you're using the latest version."); // eslint-disable-line no-alert
+        window.location = '/auth/logout';
```

Comment:
would a refresh not achieve achieve the same effect?.. Just curious on why we log them out?

Reply:
Think that is ok.